From: Joachim Bauch <jojo@struktur.de>
Date: Thu, 25 Jun 2015 15:34:10 +0200
Subject: extensions: always show component extensions in the ext list

The attached patch makes sure that component extensions are always
shown in "chrome://extensions".

Currently these are
- Bookmark Manager
- Chromium PDF Viewer
- CryptoTokenExtension

References: https://github.com/iridium-browser/iridium-browser/issues/28
---
 extensions/common/extension.cc | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/extensions/common/extension.cc b/extensions/common/extension.cc
--- a/extensions/common/extension.cc
+++ b/extensions/common/extension.cc
@@ -380,9 +380,12 @@ bool Extension::ShouldDisplayInExtensionSettings() const {
 }
 
 bool Extension::ShouldExposeViaManagementAPI() const {
+#if 0
   // Hide component extensions because they are only extensions as an
   // implementation detail of Chrome.
   return !extensions::Manifest::IsComponentLocation(location());
+#endif
+  return false;
 }
 
 Extension::ManifestData* Extension::GetManifestData(const std::string& key)
-- 
2.7.4

